tests/integration/test_ndim.py

```diff
@@ -34,6 +34,26 @@ def test_python_values(input):
     assert np.ndim(input) == num.ndim(input)
 
 
+def test_ndarray_none():
+    inp = None
+    assert np.ndim(inp) == num.ndim(inp)
+
+
+@pytest.mark.parametrize("input", ([], (), (()), ((), ()), [[]], [[], []]))
+def test_ndarray_empty(input):
+    assert np.ndim(input) == num.ndim(input)
+
+
+@pytest.mark.parametrize("input", ([[0], [1, 2], [3, 4, 5]]))
+def test_python_values_diff_dim(input):
+    assert np.ndim(input) == num.ndim(input)
+
+
+@pytest.mark.parametrize("input", ([[1, 2], [3.3, 4.4]],))
```

Comment:
This has only a single input value? Unless it is planned to add more there is no need to use `parametrize` here, just use the one input value inside the test and drop the fixture.

Reply:
fix it.